src/fn/upload-custom-translations.js

```diff
@@ -1,36 +1,90 @@
 const semver = require('semver');
 
 const environment = require('../lib/environment');
-const fs = require('../lib/sync-fs');
 const pouch = require('../lib/db');
 const getApiVersion = require('../lib/get-api-version');
 const iso639 = require('iso-639-1');
-const { warn, info } = require('../lib/log');
+const log = require('../lib/log');
+const MessageFormat = require('messageformat');
 const properties = require('properties');
 const warnUploadOverwrite = require('../lib/warn-upload-overwrite');
-
-const FILE_MATCHER = /messages-.*\.properties/;
+const {
+  checkTranslations,
+  isLanguageCodeValid,
+  TranslationException
+} = require('translation-checker');
+
+const MFORMAT = new MessageFormat('en');
+const transErrorsMsg = MFORMAT
+  .compile('There {ERRORS, plural, one{was 1 error} other{were # errors}} trying to compile');
+const transEmptyMsg = MFORMAT
+  .compile('There {EMPTIES, plural, one{was 1 empty translation} other{were # empty translations}} trying to compile');
```

Comment:
I think, now that most of the uses of `MessageFormat` have been removed from this library it would be better to remove the dependency altogether and just use hardcoded strings. This helps keep medic-conf small, fast, and simple.

Reply:
MessageFormat is still a dependency, not directly but a dependency of dependency (translation-checker). That said, maybe for the purpose of a script that may be executed inside another script, would be easier to catch the messages if they are written in only one way (eg. pipe the output of medic-conf to a grep command to check whether the were translations errors or not).

Reply:
Right.

Transitive dependencies are also better than direct dependencies, for example, if the new translation-checker package starts depending on a newer incompatible version of MessageFormat then npm will have to download two copies rather than just one. Removing the direct dependency keeps things simpler.